modules/core/src/main/java/org/apache/synapse/aspects/flow/statistics/collectors/RuntimeStatisticCollector.java

```diff
@@ -62,11 +69,21 @@
      * Initialize statistics collection when ESB starts.
      */
     public static void init() {
-        isStatisticsEnabled = Boolean.parseBoolean(
-                SynapsePropertiesLoader.getPropertyValue(StatisticsConstants.STATISTICS_ENABLE, String.valueOf(false)));
+        isMediationFlowStatisticsEnabled = Boolean.parseBoolean(
+                SynapsePropertiesLoader.getPropertyValue(
+                        StatisticsConstants.STATISTICS_ENABLE,
+                        String.valueOf(false)));
+        isOpenTracingEnabled = Boolean.parseBoolean(
+                SynapsePropertiesLoader
+                        .getPropertyValue(StatisticsConstants.OPENTRACING_ENABLE, String.valueOf(false)));
```

Comment:
can use org.apache.synapse.config.SynapsePropertiesLoader#getBooleanProperty no need to parse here since we have utility for this

Let's use that for all boolean property reads

Reply:
Done in https://github.com/wso2/wso2-synapse/pull/1466/commits/8b9ab414225c8f357d706009d81e58915e66283c